### ISSUE TITLE
a problem with the version of sd-script torch.

### ISSUE THREAD

**Issue by sports2024** [derrian-distro/LoRA_Easy_Training_Scripts#207]
I have installed it on Windows, but it seems that there is a problem with the version of torch, sd-script or something like that. It seems to be taking a lot of time during training, so please check it.No matter how many times I install it, the learning speed remains slow.

Sorry if it's strange because I'm translating from Japanese to English.

**Comment by Jelosus2** [comment]
what are you training?

**Comment by derrian-distro** [comment]
this sounds like you are using over the amount of vram you have, and as such is using RAM vs VRAM, this is incredibly slow, I suggest reducing your batch size, enabling gradient checkpointing, and using options like "full bf16" to see if it's any faster

**Comment by sports2024** [comment]
The installer.py does not specify the version when installing PyTorch, and it seems that the current stable version, 2.3.0, is installed. There are errors due to version mismatches during installation. It also appears that the versions of torch, torchvision, and xformers are not specified, so the current stable versions are installed.

I am not a programmer, so I cannot investigate this any further.
Sorry if it's strange because I'm translating from Japanese to English.

**Comment by Jelosus2** [comment]
Yea, we are aware of this issue. May I suggest you try the dev branch? It gets updates constantly and the versions are pinned.

**Comment by sports2024** [comment]
Thank you for your kindness. I tried the dev branch as my environment is Windows, but it seems it doesn't work.
Sorry if it's strange because I'm translating from Japanese to English.

**Comment by Jelosus2** [comment]
Hello, could you provide the error you get and the process you did to install the dev branch?

**Comment by sports2024** [comment]
The dev branch's backends appear to be written only for Linux, and there seems to be no code for Windows.
Sorry if it's strange because I'm translating from Japanese to English.

**Comment by Jelosus2** [comment]
Hello, any updates on this? It's been two weeks since the last reply so I'm gonna close the issue in 2 days if there's no further response. If you see this when it's closed feel free to open a new issue.
